activation/2.0/_index.md

```diff
@@ -1,35 +1,31 @@
 ---
-title: "Jakarta Activation 2.0 (under development)"
-date: 2020-mm-dd
+title: "Jakarta Activation 2.0"
+date: 2020-06-05
 summary: "Release for Jakarta EE 9"
 ---
-# Need scope statement
+Jakarta Activation defines a set of standard services to: determine the
+MIME type of an arbitrary piece of data; encapsulate access to it;
+discover the operations available on it; and instantiate the
+appropriate bean to perform the operation(s).
 
-* [Jakarta Activation 2.0 Release Record]()
+* [Jakarta Activation 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Activation 2.0 Specification Document]() (PDF)
-* [Jakarta Activation 2.0 Specification Document]() (HTML)
+* [Jakarta Activation 2.0 Specification Document](activation-spec-2.0.pdf) (PDF)
```

Comment:
Although Bill had these pretty much ready for the final ballot, I'm hesitant with making these "2.0 final" versions of the specs available without a formal review and vote.  Somehow we should indicate that these are not "final".  The header in the specs indicates "final release" and the name of the file indicates it's final.  We need to adjust something since we don't have a final vote yet.